### PR TITLE
Ability to transfer logs and artifacts to MINIO

### PR DESCRIPTION
**Pull request by Speedwag00n**
### Jira Ticket
* https://adc.luxoft.com/jira/browse/STVCIS-1563
### PURPOSE
* Ability to transfer logs and artifacts to MINIO
### EFFECT OF CHANGE
* Get necessary credentials and set them as env variables
* Add function which sends results from pipelines to MINIO
* Add calls of function which sends results from pipelines to MINIO
### TECHNICAL STEPS
* Get MINIO endpoint and set it as a env variable
* Get MINIO access and secret key and set them as env variables
### :warning: Notes for Reviewers
* Tests can be launched by choosing 'inemankov/minio_client' as a branch of pipelines repository and 'inemankov/minio_client_dev' as a branch of tests repository
### :octocat: Related PR'S
* jobs_launcher: https://github.com/luxteam/jobs_launcher/pull/92
* jobs_test_core: https://github.com/luxteam/jobs_test_core/pull/31
* jobs_test_rprviewer: https://github.com/luxteam/jobs_test_rprviewer/pull/53
### Jenkins Builds
* Blender: https://rpr.cis.luxoft.com/job/RadeonProRenderBlender2.8PluginManual/2880/
* Maya: https://rpr.cis.luxoft.com/job/RadeonProRenderMayaPluginManual/3874/
* Max: https://rpr.cis.luxoft.com/job/RadeonProRenderMaxPluginManual/1060/
* Core: https://rpr.cis.luxoft.com/job/RadeonProRenderCoreManual/783/
* RPRViewer: https://rpr.cis.luxoft.com/job/RadeonProViewerManual/893/

### CODE REVIEW

File: vars/rpr_blender28plugin_pipeline.groovy [hunk 1]
```diff
@@ -638,6 +653,18 @@ def executeBuild(String osName, Map options)
             }
         }
 
+        if (options.sendToUMS) {
+            dir('jobs_test_blender') {
+                try {
+                    checkOutBranchOrScm(options['testsBranch'], 'git@github.com:luxteam/jobs_test_blender.git')
```
Comment (epozine) [inline vars/rpr_blender28plugin_pipeline.groovy:659]: Может проще jobs_launcher скачать только?

File: vars/rpr_blender28plugin_pipeline.groovy [hunk 2]
```diff
@@ -524,6 +521,12 @@ def executeBuildWindows(Map options)
             String pluginUrl = "${BUILD_URL}/artifact/${BUILD_NAME}"
             rtp nullAction: '1', parserName: 'HTML', stableText: """<h3><a href="${pluginUrl}">[BUILD: ${BUILD_ID}] ${BUILD_NAME}</a></h3>"""
 
+            if (options.sendToUMS) {
+                dir("../../../jobs_test_blender/jobs_launcher") {
+                    sendToMINIO(options, "Windows", "..\\..\\RadeonProRenderBlenderAddon\\BlenderPkg\\.build", BUILD_NAME)
```
Comment (epozine) [inline vars/rpr_blender28plugin_pipeline.groovy:526]: В данной папке только зип архив находится?

Comment (epozine) [reply vars/rpr_blender28plugin_pipeline.groovy:526]: Стоит добавить таймаут на отправку я считаю

File: vars/rpr_blender28plugin_pipeline.groovy (hunk 1 above)
Comment (epozine) [reply vars/rpr_blender28plugin_pipeline.groovy:659]: Таймаут?

File: vars/sendToMINIO.groovy [hunk 3]
```diff
@@ -0,0 +1,28 @@
+def call(Map options, String osName, String filesPath, String pattern)
```
Comment (epozine) [inline vars/sendToMINIO.groovy:1]: Лучше таймаут наверно даже сюда добавить

**Comment by Speedwag00n** [comment]
Добавил необходимые таймауты.
Также заменил клонирование репозитория тестов на клонирование репозитория jobs_launcher на билд стадии

**Review by epozine — DISMISSED** [review]

**Review by Piromancer — APPROVED** [review]

**Review by GulDilin — APPROVED** [review]

**Pull request merged**